### PR TITLE
Master release candidate

### PR DESCRIPTION
**Pull request by zippy**
- reverts ec disabling change on master
- merges in staging
- bumps UUID to 0002 for clean hash-space

### CODE REVIEW

File: profiles/logical/hpos/default.nix [hunk 1]
```diff
@@ -191,7 +191,7 @@ in
           };
           proxy_config = {
             type = "remote_proxy_client";
-            proxy_url = "kitsune-proxy://nFCWLsuRC0X31UMv8cJxioL-lBRFQ74UQAsb8qL4XyM/kitsune-quic/h/proxy.holochain.org/p/5775/--";
+            proxy_url = "kitsune-proxy://nFCWLsuRC0X31UMv8cJxioL-lBRFQ74UQAsb8qL4XyM/kitsune-quic/h/165.22.32.11/p/5775/--";
```
Comment (alastairong) [inline profiles/logical/hpos/default.nix:194]: Which proxy are we using officially? The DO one? Why are we hardcoding the IP?

**Review by alastairong — DISMISSED** [review]
Looks good to me. Host Console login still disabled. Elemental chat should be re-enabled. One question posted.

**Review by peeech — CHANGES_REQUESTED** [review]
Staging does not represent our develop direction. This discrepancy has to be resolved before we try to merge staging to master

File: profiles/logical/hpos/default.nix (hunk 1 above)
Comment (zippy) [reply profiles/logical/hpos/default.nix:194]: DO. Encoded this way because the proxy server can't match on domain name.

File: tests/default.nix [hunk 2]
```diff
@@ -12,6 +12,7 @@ in
 
 {
   # TODO: fix/port/review these
-  # hpos-admin-api = callPackage ./hpos-admin-api {};
-  # holochain-conductor = callPackage ./holochain-conductor {};
+  # holochain = callPackage ./holochain {};
```
Comment (zo-el) [inline tests/default.nix:15]: If we want to remove the `holochain` test we should remove the test file as well

File: tests/default.nix [hunk 3]
```diff
@@ -12,6 +12,7 @@ in
 
 {
   # TODO: fix/port/review these
```
Comment (zo-el) [inline tests/default.nix:14]: We can remove this comment

**Review by zo-el — DISMISSED** [review]
LGTM. Just left a few small comments.

**Review by peeech — APPROVED** [review]
Now it looks good

**Review by Br1ght0ne — APPROVED** [review]

File: tests/default.nix (hunk 2 above)
Comment (peeech) [reply tests/default.nix:15]: that's fine for now because NixOs is lazy and won't try to evaluate

**Review by peeech — APPROVED** [review]
👍

**Review by alastairong — APPROVED** [review]

**Pull request merged**